### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by undsoft**

### CODE REVIEW

**Comment by YusukeHirao** [comment]
@undsoft 

Thank you so much.

It's good!
How did you find them?
I started to use cSpell but I don't know why it wasn't able to find them.

**Review by YusukeHirao — APPROVED** [review]

**Comment by undsoft** [comment]
Just spellcheck inspection in Webstorm.

**Comment by YusukeHirao** [comment]
Oh, nice.
So you added `.idea` to gitignore, didn't you? I see.

File: packages/@markuplint/types/src/whatwg/check-autocomplete.spec.ts [hunk 1]
```diff
@@ -369,9 +369,9 @@ test('duplicated part', () => {
 });
 
 test('typo', () => {
-	expect(check('secsion-foo name')).toStrictEqual({
+	expect(check('section-foo name')).toStrictEqual({
 		matched: false,
-		raw: 'secsion-foo',
+		raw: 'section-foo',
```
Comment (YusukeHirao) [inline packages/@markuplint/types/src/whatwg/check-autocomplete.spec.ts:374]: @undsoft 

This is purposely for the misspelling test.
Can you revert it?

**Review by YusukeHirao — CHANGES_REQUESTED** [review]

File: packages/@markuplint/types/src/whatwg/check-autocomplete.spec.ts (hunk 1 above)
Comment (undsoft) [reply packages/@markuplint/types/src/whatwg/check-autocomplete.spec.ts]: Yeah, sorry, forgot to fix this test after running.

Comment (YusukeHirao) [reply packages/@markuplint/types/src/whatwg/check-autocomplete.spec.ts]: Thank you!

**Pull request merged**